portal-cdk/portal_cdk/portal_cdk_stack.py

```diff
@@ -184,6 +212,24 @@ def __init__(
                 cache_policy=cloudfront.CachePolicy.CACHING_DISABLED,
                 response_headers_policy=cloudfront.ResponseHeadersPolicy.CORS_ALLOW_ALL_ORIGINS_WITH_PREFLIGHT_AND_SECURITY_HEADERS,
             ),
+            additional_behaviors={
+                "/error*.html": cloudfront.BehaviorOptions(
+                    origin=origins.S3Origin(error_bucket),
+                    origin_request_policy=cloudfront.OriginRequestPolicy.ALL_VIEWER_EXCEPT_HOST_HEADER,
+                    viewer_protocol_policy=cloudfront.ViewerProtocolPolicy.ALLOW_ALL,
+                    allowed_methods=cloudfront.AllowedMethods.ALLOW_GET_HEAD,
+                    cache_policy=cloudfront.CachePolicy.CACHING_DISABLED,
+                    response_headers_policy=cloudfront.ResponseHeadersPolicy.CORS_ALLOW_ALL_ORIGINS_WITH_PREFLIGHT,
+                ),
+            },
+            error_responses=[
+                cloudfront.ErrorResponse(
+                    http_status=503,
+                    response_page_path="/error.html",
+                    response_http_status=200,
```

Comment:
I think it would be good to return 503 for both 500 and 503. 200 sort of masks the issue.